server/Dockerfile

```diff
@@ -19,18 +18,12 @@ RUN find . -type f -print0 | xargs -0 dos2unix
 # Run setup
 WORKDIR /usr/src/app/server
 RUN chmod +x scripts/*.sh
-RUN make setup
 
-FROM setup AS debug
-
-ENV LOAD_TEST_DATA=true
+RUN pipenv install --deploy --ignore-pipfile || exit 1
+RUN pipenv install gunicorn
 
-EXPOSE 5002
-CMD pipenv run flask --app main --debug run --host=0.0.0.0 --port=5002 --cert=fullchain.pem --key=privkey.pem
 
 FROM setup AS prod
```

Comment:
If we decide to remove debug mode, we should adjust the README

Reply:
removed the debug mode for the server in readme